internal/util/util.go

```diff
@@ -76,3 +77,13 @@ func ResolveConfigFilePaths(config *config.MountConfig) (err error) {
 	}
 	return
 }
+
+func Stringify(input any) string {
```

Comment:
nit: We can write a doc here, which helps in understanding the method just hovering over the usage.

Reply:
done